### PR TITLE
Feature/resize widgets

### PR DESCRIPTION
**Pull request by al3xknutsen**
Video widgets should now change size based on the window width. There was quite a bit of refactoring done here, so please test and give feedback! :)

### CODE REVIEW

File: public/electron.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function quitAll() {
 // Boilerplate code - probably just quits the app when all windows are closed
 app.on('window-all-closed', () => {
   if (process.platform !== 'darwin') {
-    quitAll();
+    app.quitAll();
```
Comment (Trebua) [inline public/electron.js]: sure it is not just quitAll? because that is a function we have defined

File: src/VideoComponents/MiniMapWidget.jsx [hunk 2]
```diff
@@ -1,147 +1,82 @@
...
+      initialWidth,
+      initialHeight,
+    );
+  }
 
-  function zoomOut() {
-    if (maxDist < 1000) {
-      //maximum 1000 metres
-      setMaxDist(maxDist + 1);
+  zoomOut() {
```
Comment (Trebua) [inline src/VideoComponents/MiniMapWidget.jsx]: I think if you write const zoomOut = () => {....
then you won't need to bind like you have done above. I might be wrong, and it is not important

File: public/electron.js (hunk 1 above)
Comment (al3xknutsen) [reply public/electron.js]: The linter gave me an error without :o Where is it defined?

Comment (Trebua) [reply public/electron.js]: two lines above??

Comment (Trebua) [reply public/electron.js]: The linter does not complain when I remove app. and only keep quitAll()

**Pull request merged**